.github/workflows/python-dev.yml

```diff
@@ -1,92 +1,100 @@
-name: Python Dev
+# This file is purposely frozen. DO NOT DELETE IT
+# Unfreeze once the next Python Dev version has released beta 1
+# and both Cython and numpy support it
+# After that Python has released, migrate the workflows to the
+# posix GHA workflows/Azure pipelines and "freeze" this file by
+# commenting it out.
+# Feel free to modify this comment as necessary.
 
-on:
-  push:
-    branches:
-      - master
-      - 1.3.x
-  pull_request:
-    branches:
-      - master
-      - 1.3.x
-    paths-ignore:
-      - "doc/**"
-
-env:
-  PYTEST_WORKERS: "auto"
-  PANDAS_CI: 1
-  PATTERN: "not slow and not network and not clipboard"
-  COVERAGE: true
-  PYTEST_TARGET: pandas
-
-jobs:
-  build:
```

Comment:
Could also freeze by specifying `if: false`?

Reply:
Nit 2 other things:

`name: actions-310-dev` maybe should be `name: actions-311-dev`?

`branches: - 1.3.x` -> `- 1.4.x`